examples/tts/conf/magpietts/magpietts_lhotse_monologue.yaml

```diff
@@ -0,0 +1,213 @@
+name: MagpieTTS-EN-Lhotse
```

Comment:
Should we delete examples/tts/conf/magpietts/magpietts_lhotse.yaml? Does it still work?

Reply:
i did not test that yaml since it is tied to `MagpieTTSLhotseDataset` class which defined a different Monocut structure. Made a quick search, and did not find any places call this `MagpieTTSLhotseDataset`. We can remove it, but better confirm with @edresson.

Reply:
confirmed with @edresson, we can remove them